### ISSUE TITLE
Can this work generalize to real-world scene?

### ISSUE THREAD

**Issue by mooncake199809** [haoyu94/RoITr#13]
Nice work. I am trying to use your algorithm to register two point clouds captured in real scenes. Can we directly utilize your work to register two point clouds in real scenes without retraining?

**Comment by rjnps** [comment]
Hi @mooncake199809, have you tried this with real world data, or any other dataset not used in the paper? How is its generalization capability?

**Comment by mooncake199809** [comment]
> Hi @mooncake199809, have you tried this with real world data, or any other dataset not used in the paper? How is its generalization capability?

hellow, it is OK

**Comment by k794102577** [comment]
Can you briefly describe how you did that and what data set you used

**Comment by mooncake199809** [comment]
> Can you briefly describe how you did that and what data set you used

I use the model trained on the KITTI dataset and establish matches between two point clouds captured by ourselves.

**Comment by k794102577** [comment]
How should kitti's config.yaml be set? When I am training kitti, loss nan will appear. Could you please provide the training hyperparameters in detail
